src/openqaoa-core/qaoa_components/variational_parameters/variational_baseparams.py

```diff
@@ -0,0 +1,331 @@
+from __future__ import annotations
```

Comment:
The same goes for this files, instead of 'variational_baseparams.py', 'variational_params_converters.py', etc. I would just call them 'baseparams.py', 'converters.py', etc.

Reply:
Makes sense. We can remove variational from the name since the module name is already `variational_parameters`.

@Q-lds @vishal-ph Quick comments here would help. Thanks.

Reply:
Fine by me

Reply:
`qaoa_components/ansatz_constructor` already has `baseparams.py`. Might lead to confusion

Reply:
I think its fine as long as we don't expose `baseparams.py` in `qaoa_components`.

Do you think `qaoa_components.ansatz_constructor.baseparams` and `qaoa_components.variational_parameters.baseparams` would still be too confusing?

Reply:
Keeping the name